test/analysis_pp_hTo4l_cfg.py

```diff
@@ -0,0 +1,292 @@
+import os
```

Comment:
nice and clean cfg. 
You might just want to add more inline comments as in the other example cfgs, but this your choice.